src/components/RegisterForm/components/Fields.js

```diff
@@ -0,0 +1,334 @@
+import { Component } from 'pet-dex-utilities';
+import Dropdown from '../../Dropdown/index';
+import Button from '../../Button/index';
+import TextInput from '../../TextInput/index';

```

Comment:
esse index no final não deveria ser nescessario

```suggestion
import Dropdown from '../../Dropdown';
import Button from '../../Button';
import TextInput from '../../TextInput';

```